src/options.json

```diff
@@ -5439,7 +5439,7 @@
         "title": "Don't join curly brackets",
         "type": "boolean",
         "default": true,
-        "descriotion": "",
```

Comment:
Found it: https://github.com/Glavin001/atom-beautify/blame/c881bdf656f6ab3daf5516209d3960f97ba1ab83/src/languages/nginx.coffee#L41

Sigh. Looks like I should add spell check along with the linter 😝 .